learn/useCases/maintenanceScheduling.adoc

```diff
@@ -0,0 +1,41 @@
+= Maintenance scheduling
+:awestruct-description: Use OptaPlanner (java™, open source) to optimize assigning tasks to employees with different skills and affinities.
+:awestruct-layout: useCaseBase
+:awestruct-priority: 1.0
+:awestruct-related_tag: maintenance scheduling
+:showtitle:
+
+Machinery and other types of equipment in use (such as elevators, escalators, airplanes, trains, ...)
+tend to deteriorate over time, requiring regular maintenance.
+Depending on how often each machine requires care,
+_maintenance scheduling_ decides when and which skilled engineer visits each machine.
+Optimizing this planning problem with a constraint solver AI saves costs
+by reducing both too early and too late maintenance,
+while adhering to other constraints such as skill requirements, employee availability
+and the geographic location of machines.
+
+// TODO Maintenance scheduling value proposition image
+
+== What is OptaPlanner?
+
+OptaPlanner is the leading Open Source Java™ AI constraint solver
+to optimize maintenance scheduling,
+adhering to skill, capacity, SLA's and other constraints.
+
+OptaPlanner is *a lightweight, embeddable planning engine*.
+It enables normal Java™ programmers to solve optimization problems efficiently.
```

Comment:
The word "normal" might be interpreted in a way we probably don't want our users to interpret it :)

Let's just say "Java programmers".

Reply:
Interesting point.
What I really wanted to say is "blue-collar Java programmers", or "average" or "non-elite". But none of those are better, obviously :)
 
how about "any Java programmer"?

Reply:
BTW, we have this phrase on every use case page and the front page... (yes DRY, I know :)

@triceo ^